dspace-api/src/main/java/org/dspace/embargo/EmbargoManager.java

```diff
@@ -460,5 +460,18 @@ private static DCDate recoverEmbargoDate(Item item) {
         }
         return liftDate;
     }
-
+    
+    
+	static public ItemIterator getEmbargoedItems(Context context) 
```

Comment:
> java.lang.IllegalArgumentException: No such metadata schema: SCHEMA

this is a configuration issue, somewhere in your config you have embargo.field.{terms,lift}=SCHEMA.ELEMENT.QUALIFIER

you should still see an exception but the message should be more verbose now...